.editorconfig

```diff
@@ -10,4 +10,7 @@ end_of_line = lf
 indent_size = 2
 indent_style = tab
 insert_final_newline = true
-trim_trailing_whitespace = true
\ No newline at end of file
+trim_trailing_whitespace = true
+
+[*.json]
+insert_final_newline = false
```

Comment:
I saw in a different review that someone was asked to remove the trailing space from the config file. Is this something we do? If so I put this in here so editors don't add one by default.

If it's not a thing I will remove this :)

Reply:
I don't know whether that is a thing, but I'm leery of adding changes that are related to the ticket.

Reply:
You're not sure what is a thing? The convention that we remove spaces at the end of config files? If that's the case that makes me happy... intelliJ adds a spaces at the end if it's true. If it's not a convention we have, that makes me happy and I'll remove it and I'll be less afraid of checking in whitespace only files by mistake.

Reply:
if this thing prevents that constant package.json final newline, it would be great!

Reply:
It does yes :)